lib/tag-chip.jsx

```diff
@@ -1,29 +1,14 @@
-import React from 'react'
-import classNames from 'classnames'
-
-export default React.createClass( {
-
-	getDefaultProps: function() {
-		return {
-			tag: '',
-			selected: false,
-			onSelect: function() {}
-		};
-	},
-
-	onClick: function( e ) {
-		e.preventDefault();
-		this.props.onSelect( e );
-	},
-
-	render: function() {
-		return (
-			<a
-				className={classNames( 'tag-chip', { selected: this.props.selected } )}
-				href="#"
-				onClick={this.onClick}>
-				{this.props.tag}
-			</a>
-		)
-	}
-} );
+import React from 'react';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+
+export const TagChip = ( { onSelect = noop, selected, tag: tagName } ) => (
+	<div
+		className={ classNames( 'tag-chip', { selected } ) }
+		onClick={ onSelect }
+	>
```

Comment:
Not sure why this was previously and <a />, but I couldn't think of any reason to keep it like that